packages/admin/admin/tsconfig.json

```diff
@@ -4,5 +4,6 @@
         "outDir": "lib",
         "rootDir": "src"
     },
-    "include": ["./src"]
+    "include": ["./src"],
+    "exclude": ["src/**/*.spec.ts", "src/**/*.test.ts"]
```

Comment:
@nsams Is that okay? I added it directly to the `tsconfig.json` because in `cms-admin` we also don't have an extra `tsconfig.build.json`

Reply:
we have in next

And there is a reason for that, see PR description https://github.com/vivid-planet/comet/pull/1141

maybe do the whole testing setup in next only?

Reply:
Okay, I wasn't aware of that. But I can still add it to main, right?
I need the fix in #1302 in main and I'd like to have the tests as well.

Reply:
yes, you can also do that.

(my idea was to avoid possibly breaking changes in main, but this has proven to be working)


> and I'd like to have the tests as well.

would be perfectly fine to have the tests in next only (the fix itself you need in main of course)